invenio_jobs/alembic/1f896f6990b8_update_jobs_module_table_names.py

```diff
@@ -0,0 +1,59 @@
+#
+# This file is part of Invenio.
+# Copyright (C) 2016-2018 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Update jobs module table names."""
+
+import sqlalchemy as sa
+import sqlalchemy_utils
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "1f896f6990b8"
+down_revision = "356496a01197"
+branch_labels = ()
+depends_on = None
+
+
+def upgrade():
```

Comment:
tested locally on zenodo instance, upgrades and downgrades nicely, there is a data loss though, since we drop defualt args, to be discussed